staging/vhost-device-console/README.md

```diff
@@ -0,0 +1,131 @@
+# vhost-device-console - Console emulation backend daemon
+
+## Description
+
+This program is a vhost-user backend that emulates a VirtIO Console device.
+The device's binary takes as parameter a socket, a socket number which is the
+number of connections, commonly used across all vhost-devices to communicate
+with the vhost-user frontend devices, and the backend type "nested" or
+"network".
+
+By using the "nested" backend, vhost-device-console, as soon as, the guest
+is booted, will print the login prompt into the same terminal and gives
+the ability to insert characters or certain keys as in a regular
+terminal.
+
+By using "network" backend, vhost-device-console, as soon as the guests
+is booted, creates a local server in the given localhost port, and gives
```

Comment:
Which kind of connection? tcp?

Reply:
Yes that's a TCP connection. The device has a TCPlisener when started with `network` option, and waits for a TCP connection.
I describe that later in the README file but I have also added a clarification where you pointed out.